### PR TITLE
add spiegel-de subdomain

### PR DESCRIPTION
**Pull request by numismatika**
also re-enables the ruleset

### CODE REVIEW

File: src/chrome/content/rules/Spiegel.xml [hunk 1]
```diff
@@ -110,27 +62,12 @@ Fetch error: http://ophirum.spiegel.de/ => https://ophirum.spiegel.de/: (51, "SS
...
-
-	<securecookie host="^(?:abo|count|magazin|ophirum)\.spiegel\.de$" name=".+" />
-
-
-	<!--	- Cert only matches www
-		- Unsupported paths redirect to http
-		- cdn\d?: Akamai
-				-->
-	<rule from="^http://cdn\d?\.spiegel\.de/"
-		to="https://www.spiegel.de/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Spiegel.xml]: Why did you remove this?

File: src/chrome/content/rules/Spiegel.xml [hunk 2]
```diff
@@ -110,27 +62,12 @@ Fetch error: http://ophirum.spiegel.de/ => https://ophirum.spiegel.de/: (51, "SS
 			<test url="http://www.spiegel.de/pics/43/0,1020,308043,00.jpg" />
 			<test url="http://www.spiegel.de/static/sys/dimensionspixel.gif" />
 
+	<securecookie host=".+" name=".+" />
```
Comment (J0WI) [inline src/chrome/content/rules/Spiegel.xml]: This is quite dangerous since we exclude major party of the site.

File: src/chrome/content/rules/Spiegel.xml (hunk 1 above)
Comment (numismatika) [reply src/chrome/content/rules/Spiegel.xml]: Because the first step when i am cleaning up old rulesets is deleting all special rewrites to see if it works without them. Also there are no test urls to be found , neither in google nor on the spiegel website.

Comment (J0WI) [reply src/chrome/content/rules/Spiegel.xml]: I can find many examples when browsing on the site. With the rule they get secured, but without it you can see the requests to `cdn([1-4])?`.
